README.md

```diff
@@ -1 +1,91 @@
-# EvenStoreDB Cloud CLI
\ No newline at end of file
+# EvenStoreDB Cloud CLI
+
+`esc` in a command-line tool that allows to access EventStoreDB Cloud API in the confort of your terminal.
+
+## Authentication
+
+If you use `esc` for the first time, you need to generate a token first. By default, `esc` will interactively
+ask your email and password to create your token.
+
+If you prefer to have a non-interactive way to create your token, enter that command instead:
+
+```
+esc access tokens create --email <email> --unsafe-password <password>
+```
+
+`esc` will refresh your token automatically without you needing to do anything. Rest assured that
+`esc` doesn't store your password in your system.
+
+## Implicit parameters
```

Comment:
Required parameters?

Reply:
I guess they are implicit because they can be stored in the profile and therefore not provided explicitly for each call.

Reply:
Is there a better to express it? I wanted to say what @alexeyzimarev said.

Reply:
On second read I agree with you both, implicit is better